lighthouse-core/audits/screenshot-thumbnails.js

```diff
@@ -30,8 +32,8 @@ class ScreenshotThumbnails extends Audit {
    * Scales down an image to THUMBNAIL_WIDTH using nearest neighbor for speed, maintains aspect
    * ratio of the original thumbnail.
    *
-   * @param {{width: number, height: number, data: !Array<number>}} imageData
-   * @return {{width: number, height: number, data: !Array<number>}}
```

Comment:
do we want to reference the speedlineoutput.frames type instead?

Reply:
SG, done